apps/backend/src/libs/helpers/transform-to-snake-case/transform-to-snake-case.helper.ts

```diff
@@ -0,0 +1,7 @@
+import snakeCase from "lodash/snakeCase.js";
+
+const transformToSnakeCase = (name: string): string => {
```

Comment:
use https://www.npmjs.com/package/change-case this library instead. install it in the shared folder and create a wrapper around this function